data/scripts/bin/stop-local-registry.sh.template

```diff
@@ -0,0 +1,13 @@
+#!/usr/bin/env bash
+
+# Wait for cluster to be ready
+kubeconfig=/etc/kubernetes/static-pod-resources/kube-apiserver-certs/secrets/node-kubeconfigs/localhost.kubeconfig
+until [ "$(sudo oc get clusterversion --kubeconfig "${kubeconfig}" -o jsonpath='{.items[*].status.conditions[?(@.type=="Available")].status}')" == "True" ];
```

Comment:
just making sure here (bash can sometimes be misleading): will this command ever timeout if the condition is not met?

Reply:
It won't timeout, the service should stay alive. I.e. in case the cluster is failing there's nothing we can do automatically, so there's no real point for a timeout. Also, if the cluster can be fixed manually, this service should stay active to detect it.